### PR TITLE
Fix broken link in vlan group template

### PR DESCRIPTION
**Pull request by johannwagner**
# Closes: #2573 
# What's Changed
- Changing a link from an id to a slug to match filter form

### CODE REVIEW

**Comment by johannwagner** [comment]
This actually does introduce a different problem. The implemented solution does filter correctly but does not show the filter in the sidebar. I will provide a different implementation.

**Comment by johannwagner** [comment]
This now shows the selected VLAN group in the filter form on the right side.

File: changes/2573.fixed [hunk 1]
```diff
@@ -0,0 +1 @@
+Fixing link to VLAN list to match filter form.
```
Comment (glennmatthews) [inline changes/2573.fixed:1]: Perhaps?

```suggestion
Fixed link from VLAN group detail view to filtered VLAN list.
```

**Review by glennmatthews — APPROVED** [review]
Thank you!

**Pull request merged**